pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[tool.poetry]
```

Comment:
nit: personally don't have experience with poetry, used hatch instead. But I guess the idea is still more or less the same

Reply:
This is autogenerated by poetry and has similarities.